cluster/juju/layers/kubernetes-master/templates/heapster-controller.yaml

```diff
@@ -0,0 +1,138 @@
+{% set base_metrics_memory = "140Mi" -%}
```

Comment:
This looks like a lot of variables. Did you have to deviate from the original template much here?